pkg/kube/client.go

```diff
@@ -152,7 +152,7 @@ func (c *Client) Get(namespace string, reader io.Reader) (string, error) {
 		return "", err
 	}
 	err = perform(c, namespace, infos, func(info *resource.Info) error {
-		log.Printf("Doing get for: '%s'", info.Name)
+		log.Printf("Doing get for %s: '%s'", info.Mapping.GroupVersionKind.Kind, info.Name)
```

Comment:
I didn't add the single quotation mark I just added the object type to to the print statement.